File: importado-do-antigo/ALGORITMOS-E-ESTRUTURAS-DE-DADOS-II/teo/U0-nivelamento/0-Intro_Lang_Prog/aula5/comand_IF.java
import java.util.Scanner;

public class comand_IF {
    public static void main(String[] args) {
        int num;

        Scanner teclado = new Scanner(System.in);
            System.out.print("Digite um número: ");
            num = teclado.nextInt();

            if (num < 0) {
                System.out.printf("\nO número %d é negativo!", num);
            }
            else{
                //Considerar 0 como nulo.
                if (num == 0) {
                    System.out.printf("\nO número %d é nulo!", num);
                }
                else{
                    System.out.printf("\nO número %d é positivo!", num);
                }
            }
        teclado.close();
    }
}

File: importado-do-antigo/ALGORITMOS-E-ESTRUTURAS-DE-DADOS-II/teo/U0-nivelamento/0-Intro_Lang_Prog/aula5/comand_IF.cpp
#include <iostream>

using namespace std;

int main(int argc, char const *argv[]){
    int num;
    
    cout << "Digite um número: ";
    cin num;

    if (num < 0){
        cout << "O número " << num << " é negativo!";
    }
    else{       
        //Considerar que 0 é nulo.
        if (num == 0){
            cout << "O número " << num << " é positivo!";
        }
        else{
            cout << "O número " << num << " é positivo!";
        }
        
    }
    
    
    return 0;
}
